superset-frontend/src/dashboard/components/gridComponents/Row.jsx

```diff
@@ -69,6 +81,37 @@ class Row extends React.PureComponent {
       'background',
     );
     this.handleChangeFocus = this.handleChangeFocus.bind(this);
+
+    this.containerRef = React.createRef();
+    this.observer = null;
+  }
+
+  componentDidMount() {
+    if (
+      isDashboardVirtualizationEnabled(this.props.dashboardVirtualizationMode)
+    ) {
+      this.observer = new IntersectionObserver(
```

Comment:
Should we [disconnect](https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/disconnect) and [unobserve](https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/unobserve) on unmount? Just to reduce the  chance of memory leak.

Reply:
Done